docs/user_guide.md

```diff
@@ -0,0 +1,96 @@
+#User Guide
+##Overview
+This guide takes you through the fundamentals of using Harbor. You'll learn how to use Harbor to:  
+
+* Manage your projects.
+* Manage the members of a project.
+* Search projects and repositories.
+* Manage Harbor system if you are the system administrator.
+* Pull and push images using Docker client.
+
+
+##Role Based Access Control
+RBAC (Role Based Access Control) is provided in Harbor and there are four roles with different privileges:  
+
+* **Guest**: Guest has read-only privilege for a specified project.
+* **Developer**: Developer has read and write privileges for a project.
+* **ProjectAdmin**: When create a new project, you will be assigned the "ProjectAdmin" role to the project. Besides read-write privileges, the "ProjectAdmin" also has some management privileges, such as adding and removing members.
+* **SysAdmin**: "SysAdmin" has the most privileges. In addition to the privileges mentioned above, "SysAdmin" can also list all projects, set an ordinary user as administrator and delete users. The public project "library" is also owned by the administrator.  
+
+##Managing projects
+A project in Harbor is a namespace that role based access control is applied to it and the repositories in it. Two types of projects are provided: 
+
+* **Public**: All users have the read privilege to a public project, it's convenient for you to share some repositories with others in this way.
+* **Private**: A private project can only been accessed by its members.
+
+You can create a project after you signing in. Enable the "Public project" checkbox will make this project public.  
+
+![create project](img/create_project.png)  
+
+After the project being created, you can browse the repositories, members and access logs using the tabs in the left.  

```

Comment:
+After the project is created, you can browser repositories, users and access logs using the navigation column on the left.